api/handler/handler.go

```diff
@@ -0,0 +1,47 @@
+package handler
+
+import (
+	"encoding/json"
+	"net/http"
+	"spd-lookup/api/data"
+)
+
+// Handler is the struct for route handler functions
+type Handler struct {
+	db data.DatabaseInterface
+}
+
+// NewHandler is the constructor for the handler
+func NewHandler() *Handler {
+	return &Handler{
+		db: data.NewClient(),
+	}
+}
+
+// Ping pong :^)
+func (h *Handler) Ping(w http.ResponseWriter, r *http.Request) {
+	w.Write([]byte("🏓 P O N G 🏓"))
```

Comment:
Please do not delete this at any point going forward, it continues to be super helpful when testing infra/whether the service is up haha